weepickle-implicits/src/main/scala-3/com/rallyhealth/weepickle/v1/implicits/macros.scala

```diff
@@ -56,7 +56,7 @@ def getDefaultParamsImpl[T](using Quotes, Type[T]): Expr[String => Option[() =>
     val namesExpr: Expr[List[String]] = Expr.ofList((names ++ assumeDefaultNoneNames).map(Expr(_)))
     val identsExpr: Expr[List[() => AnyRef]] = Expr.ofList(
       idents.map(_.asExpr).map(i => '{ () => $i.asInstanceOf[AnyRef] }) ++
-      assumeDefaultNoneNames.map(_ => '{ () => None })
+      assumeDefaultNoneNames.map(_ => '{ () => scala.None })
```

Comment:
is this one actually necessary? Seems like `None` should be unambiguous in this context.

Reply:
I will resort to @russellremple on this one... I don't actually know if there are scala3 tests and if I can add something to test if it's necessary.

Reply:
The `ExampleTests` addition you made covers both scala 2 and 3. If removed, does the test pass?

Reply:
You are right, it isn't needed for Scala 3. Can you explain why "`None` should be unambiguous in this context"? You are 100% correct, I just don't understand.